lib/sarif/sarif_report.rb

```diff
@@ -0,0 +1,55 @@
+require 'json'
+require 'json-schema'
+require_relative './base_sarif'
+module Sarif
+  # Class for generating sarif reports
+  class SarifReport
+    class SarifInvalidFormatError < StandardError; end
+
+    SARIF_VERSION = "2.1.0".freeze
+
+    SARIF_SCHEMA = "https://schemastore.azurewebsites.net/schemas"\
```

Comment:
Any reason for this site vs the SARIF site itself for sourcing the schema? 
http://docs.oasis-open.org/sarif/sarif/v2.0/csprd01/schemas/sarif-schema.json